packages/browser/src/types.ts

```diff
@@ -968,9 +968,10 @@ export interface PostHogConfig {
 
     /**
      * PREVIEW - MAY CHANGE WITHOUT WARNING - DO NOT USE IN PRODUCTION
-     * Whether to wrap fetch and add tracing headers to the request
+     * A list of hostnames for which to inject PostHog tracing headers to all requests
+     * (X-POSTHOG-DISTINCT-ID, X-POSTHOG-SESSION-ID, X-POSTHOG-WINDOW-ID)
      * */
-    __add_tracing_headers?: boolean
+    __add_tracing_headers?: string[]
```

Comment:
we could be nicer and accept `boolean | string[]` 
although we do warn it might change so 🙈